### ISSUE TITLE
Small code optimization

### ISSUE THREAD

**Issue by kek-Sec** [DEMON1A/GmailBruterV2#6]
Adding a break to not loop all passwords if SMTP server disconnected so it prints only once

**Comment by DEMON1A** [comment]
Thanks, @kek-Sec, Yeah I can see that the program keeps printing this message and keeps trying to connect over again without stop. I should break the loop here and when the program finds good credits. instead of keep trying to connect. that increases the ban chances and disallows further login processes into this Gmail account since google will send him an alert.

**Comment by DEMON1A** [comment]
I did test it locally and I can see that it's working. thanks again for that your issue allowed me to find a flow in Gmail SMTP would allow me to validate if passwords are valid or not using Bad SMTP attempts. but the real issue here is how to login without google validation. gonna work on a new update soon. if you can see there's still something that needs to be edited please submit a new issue with it. I will close this issue later.

**Comment by kek-Sec** [comment]
Glad I could help looking forward to the update, I have some bypass ideas
